src/simplewallet/simplewallet.cpp

```diff
@@ -4458,41 +4458,20 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
 		fail_msg_writer() << tr("wrong number of arguments");
 		return true;
 	}
-
-	std::vector<uint8_t> extra;
-	bool payment_id_seen = false;
+	
+	crypto::uniform_payment_id payment_id;
+	payment_id.zero = (-1);
```

Comment:
since we have a default constructor and initialize it as invalid this line can be removed